src/process.cpp

```diff
@@ -262,6 +283,12 @@ Process *getProcess(unsigned long inode, const char *devicename) {
   if (proc != NULL)
     return proc;
 
+  if ( !(pidsToWatch.empty()) && pidsToWatch.find(node->pid) == pidsToWatch.end() ) {    
```

Comment:
(not sure if I'm a fan of the additional spaces here, but since we don't enforce/autoformat in this project I'm OK with it ;) )

Reply:
Maybe, add a formatter would be a nice new issue! 😁